packages/gatsby-image/src/index.js

```diff
@@ -1,18 +1,40 @@
 import React from "react"
 import PropTypes from "prop-types"
 
-// Handle legacy names for image queries.
-const convertProps = props => {
-  let convertedProps = { ...props }
-  if (convertedProps.resolutions) {
-    convertedProps.fixed = convertedProps.resolutions
+const logDeprecationNotice = (prop, replacement) => {
```

Comment:
I figured it could be used for `resolutions` and `sizes` too? Those aren't officially supported anymore are they? There isn't reason to keep the around afaik, so should they also raise deprecation notices?

The utility function is probably better suited out of `gatsby-image` but I'm not sure where it should go or if it'd be useful to any other packages.